docs/Coding-Standards/React-Components.stories.mdx

```diff
@@ -259,6 +260,14 @@ And all components meeting our standards will from now on will be maintained in
 
 Where possible we want to apply standardisation as an ‘opportunistic refactor’. As we look to use existing components to deliver a feature, we commit to rebuilding those components or porting them across applying new best practices. Through doing this we gradually improve the consistency and standards of our components rather then embarking on a unrealistic drive to rebuild all components to a new standard. The necessity of delivering a feature keeps scope under control while the standards maintain the quality of the code for that feature.
 
+Since it will be not be feasible to migrate all components to the new standard, sometimes we will need to make a judgment call, on a case-by-case basis, about when or whether to apply standardisation.
+
+As a rule of thumb, the following criteria could be used to help make this decision:
+- The component requires significant modification as part of the feature delivery
```

Comment:
Maybe explain the rationale for this, the component is undergoing significant change anyway, we can opportunistically refactor it now and make it more understandable for future maintainers?